packages/flutter/lib/src/material/slider.dart

```diff
@@ -434,6 +454,15 @@ class _SliderState extends State<Slider> with TickerProviderStateMixin {
     );
     enableController.value = widget.onChanged != null ? 1.0 : 0.0;
     positionController.value = _unlerp(widget.value);
+    _shortcutMap = <LogicalKeySet, Intent>{
+      LogicalKeySet(LogicalKeyboardKey.shift, LogicalKeyboardKey.arrowUp): const _AdjustSliderIntent.increment(),
+      LogicalKeySet(LogicalKeyboardKey.shift, LogicalKeyboardKey.arrowDown): const _AdjustSliderIntent.decrement(),
+      LogicalKeySet(LogicalKeyboardKey.shift, LogicalKeyboardKey.arrowLeft): const _AdjustSliderIntent.decrement(),
```

Comment:
Should L/R arrow keys behave oppositely in RTL locales? not sure if there is a standard for this or not.

Actually does the slider itself get mirrored in RTL?

Reply:
Yes RTL left should be increment and right should be decrement, Updated these controls